relay-general/src/pii/convert.rs

```diff
@@ -39,7 +31,8 @@ pub fn to_pii_config(
     }
 
     if datascrubbing_config.scrub_ip_addresses {
-        applications.insert(KNOWN_IP_FIELDS.clone(), vec!["@anything:remove".to_owned()]);
+        let wildcard = SelectorSpec::Path(vec![SelectorPathItem::DeepWildcard]);
+        applications.insert(wildcard, vec!["@ip:replace".to_owned()]);
```

Comment:
The question is do we want to keep this consistent with old behaviour and just remove them or actually switch to replacing all the IPs as you did here.